### PR TITLE
Fix exchange rate & fee schedule APIs

### LINKED ISSUE #3958: Network fees call fails due to exchange rate throwing internal error

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:
* Fix exchange rate & fee schedule APIs by returning raw buffer
* Fix fee schedule & throttles persistence by treating partial success response codes as success
* Fix `s3client.test.js` `valid config with endpointOverride` always failing locally

**Related issue(s)**:

Fixes #3958

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-rest/__tests__/specs/network-fees-02-order.spec.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
       {
         "consensus_timestamp": 1234567890900800100,
         "entity_id": 111,
-        "file_data": "0a280a0a08541a061a0440a8953a0a0a08061a061a0440a8aa330a0a0",
+        "file_data": "0a280a0a08541a061a0440a8953a0a0a08061a061a0440a8aa330a0a",
```
Comment (steven-sheehy) [inline hedera-mirror-rest/__tests__/specs/network-fees-02-order.spec.json:14]: Two odd length file chunks couldn't be properly loaded as hex. The fix is to shift one character on one to the other so they're both even length again.

File: hedera-mirror-common/src/main/java/com/hedera/mirror/common/domain/transaction/RecordItem.java [hunk 2]
```diff
@@ -178,7 +178,10 @@ public SignatureMap getSignatureMap() {
     }
 
     public boolean isSuccessful() {
-        return record.getReceipt().getStatus() == ResponseCodeEnum.SUCCESS;
+        var status = record.getReceipt().getStatus();
```
Comment (Nana-EC) [inline hedera-mirror-common/src/main/java/com/hedera/mirror/common/domain/transaction/RecordItem.java:181]: Does this need a migration?
Presumably there are some transactions that were incorrectly marked as a failure that we may have to go back and update.

File: hedera-mirror-rest/__tests__/service/fileDataService.test.js [hunk 3]
```diff
@@ -37,25 +37,25 @@ const files = [
   {
     consensus_timestamp: 1,
     entity_id: 112,
-    file_data: '0a1008b0ea0110cac1181a0608a0a1d09306121008b0ea0110e18e191a0608b0bdd09306',
+    file_data: Buffer.from('0a1008b0ea0110cac1181a0608a0a1d09306121008b0ea0110e18e191a0608b0bdd09306', 'hex'),
```
Comment (Nana-EC) [inline hedera-mirror-rest/__tests__/service/fileDataService.test.js:40]: nit: use a shared method to 
```
const decode = (contents) => {
  return Buffer.from(contents, 'hex');
}
```

then

```suggestion
    file_data: decode('0a1008b0ea0110cac1181a0608a0a1d09306121008b0ea0110e18e191a0608b0bdd09306'),
```

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG.
Suggestion and potential missing migration

File: hedera-mirror-common/src/main/java/com/hedera/mirror/common/domain/transaction/RecordItem.java (hunk 2 above)
Comment (Nana-EC) [reply hedera-mirror-common/src/main/java/com/hedera/mirror/common/domain/transaction/RecordItem.java:181]: Scratch that. I was mixing this up with the status value used to set Result

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Unblocking my previous change request

**Review by Nana-EC — APPROVED** [review]
Actually unblocking my previous change request

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**